src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amountRent = 40;
+
+  if (days < 3) {
+    return amountRent * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (amountRent * days) - 20;
+  }
+
+  if (days >= 7) {
+    return (amountRent * days) - 50;
+  }
```

Comment:
you can do it shorter like:
1. first check  -  if (days >= 7) {
2. second check - if (days >= 3) 
3. just return (for everything than less 3)

![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/91946036/467f31cc-affe-4c67-8806-d285b8df3970)